e2e/rbd_helper.go

```diff
@@ -96,25 +96,6 @@ func createRadosNamespace(f *framework.Framework) error {
 			return fmt.Errorf("error creating rbd namespace %v", stdErr)
 		}
 	}
-	stdOut, stdErr, err = execCommandInToolBoxPod(f,
-		fmt.Sprintf("rbd namespace ls --pool=%s", rbdTopologyPool), rookNamespace)
```

Comment:
There are two namespaces created here, `defaultRBDPool` and `rbdTopologyPool`. You could clean the code up by creating a separate function to create a single namespace.

Reply:
I think I made a mistake in my first commit! What is the point of two different pools in testing?

Reply:
Ceph supports "Topology Aware Provisioning" and that requires testing with different pools, see #816 for further details.